src/main/java/org/isf/utils/jobjects/PaginatedTableDecoratorSimple.java

```diff
@@ -0,0 +1,213 @@
+package org.isf.utils.jobjects;
+import javax.swing.*;
```

Comment:
The project does not use package imports; all imports are explicitly enumerated

Reply:
Done, thanks!